apps/namadillo/src/App/Transfer/TransferModule.tsx

```diff
@@ -471,6 +472,7 @@ export const TransferModule = ({
             )}
             isShieldedAddress={destination.isShieldedAddress}
             isShieldedTx={isShieldedTx}
+            setIsDisabled={setIsDisabled}
```

Comment:
Feels kinda weird to prop drill just for the CustomAddressForm here.

But I didn't see any other way to disable the `Submit` from an address error.

Reply:
I don't think it should be a blocker, but in case you want to explore alternatives here, maybe you could try to create a new option for `ValidationResult` that handle this case. It should disable the submit button

![Screenshot 2025-03-31 at 17 13 02](https://github.com/user-attachments/assets/63c1be85-7213-4449-bd54-817dc682ae72)

Reply:
I updated to use this and it's wayyyy better...so much less surface area of the code touched.

Thank you for the suggestion.

Reply:
<img width="520" alt="Screenshot 2025-04-01 at 12 18 53 PM" src="https://github.com/user-attachments/assets/d1cdb93e-4f7e-48fc-81cf-07e026911450" />